### PR TITLE
Update the relevant notebook with a stand alone dataset download

### PR DESCRIPTION
**Pull request by Idan-BenAmi**
## Pull Request Description:
Update the Keras PTQ notebook with a dataset download and make additional minor modifications.

## Checklist before requesting a review:
- [x] I set the appropriate labels on the pull request.
- [x] I have added/updated the release note draft (if necessary).
- [x] I have updated the documentation to reflect my changes (if necessary).
- [x] All function and files are well documented. 
- [x] All function and classes have type hints.
- [x] There is a licenses in all file.
- [x] The function and variable names are informative. 
- [x] I have checked for code duplications.
- [x] I have added new unittest (if necessary).

### CODE REVIEW

File: tutorials/notebooks/example_keras_imagenet.ipynb [hunk 1]
```diff
@@ -1,482 +1,515 @@
...
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "699be4fd-d382-4eec-9d3f-e2e85cfb1762",
+   "metadata": {
+    "id": "699be4fd-d382-4eec-9d3f-e2e85cfb1762"
+   },
+   "source": [
+    "This tutorial shows how to quantize a pre-trained model using the Model Compression Toolkit (MCT). We will do so by giving an example of MCT's post-training quantization. As we will see, post-training quantization is a low complexity yet effective quantization method. In this example, we will quantize the model and evaluate the accuracy before and after quantization."
```
Comment (elad-c) [inline tutorials/notebooks/example_keras_imagenet.ipynb:40]: maybe call it "quantization scheme" not "method"

File: tutorials/notebooks/example_keras_imagenet.ipynb [hunk 2]
```diff
@@ -1,482 +1,515 @@
...
+   "id": "9c0e9543-d356-412f-acf1-c2ecad553e06",
+   "metadata": {
+    "id": "9c0e9543-d356-412f-acf1-c2ecad553e06"
+   },
+   "source": [
+    "In this tutorial we will cover:\n",
+    "\n",
+    "1. Post-Training Quantization using MCT.\n",
+    "2. Loading and preprocessing ImageNet's validation dataset.\n",
+    "3. Loading and preprocessing an unlabeled representative dataset from the ImageNet trainset.\n",
```
Comment (elad-c) [inline tutorials/notebooks/example_keras_imagenet.ipynb:64]: Add comment the you work on validation set for convenience

File: tutorials/notebooks/example_keras_imagenet.ipynb [hunk 3]
```diff
@@ -1,482 +1,515 @@
...
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "8a8b486a-ca39-45d9-8699-f7116b0414c9",
+   "metadata": {
+    "id": "8a8b486a-ca39-45d9-8699-f7116b0414c9"
+   },
+   "source": [
+    "For the sake of demonstration, let's set few MCT parameters. Note that you can skip this part in case you want to use the default settings."
```
Comment (elad-c) [inline tutorials/notebooks/example_keras_imagenet.ipynb:247]: a few

File: tutorials/notebooks/example_keras_imagenet.ipynb [hunk 4]
```diff
@@ -1,482 +1,515 @@
...
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "outputs": [],
+   "source": [
+    "# Specify the target platform capability (TPC)\n",
+    "target_platform_cap = mct.get_target_platform_capabilities('tensorflow', 'default')\n",
```
Comment (elad-c) [inline tutorials/notebooks/example_keras_imagenet.ipynb:256]: use the imx500v1 TPC (same as the effdet notebook)

File: tutorials/notebooks/example_keras_imagenet.ipynb [hunk 5]
```diff
@@ -1,482 +1,515 @@
...
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "33f8373a-82a5-4b97-9a10-25ee2341d148",
+   "metadata": {
+    "id": "33f8373a-82a5-4b97-9a10-25ee2341d148"
+   },
+   "outputs": [],
+   "source": [
+    "quantized_model, quantization_info = mct.ptq.keras_post_training_quantization_experimental(float_model, representative_dataset_gen, core_config=ptq_config)"
```
Comment (elad-c) [inline tutorials/notebooks/example_keras_imagenet.ipynb:291]: you defined "target_platform_cap" but didn't use it for the API call

File: tutorials/notebooks/example_keras_imagenet.ipynb [hunk 6]
```diff
@@ -1,482 +1,515 @@
...
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ebfbb4de-5b6e-4732-83d3-a21e96cdd866",
+   "metadata": {
+    "id": "ebfbb4de-5b6e-4732-83d3-a21e96cdd866"
+   },
+   "source": [
+    "You can see that we got a very small degradation with a compression rate of x4 !"
```
Comment (elad-c) [inline tutorials/notebooks/example_keras_imagenet.ipynb:414]: does model.evaluate print the result? maybe print both results in the same line for easy comparison

File: tutorials/notebooks/example_keras_imagenet.ipynb [hunk 7]
```diff
@@ -1,482 +1,515 @@
...
+    "id": "6YjIdiRRjgkL"
+   },
+   "id": "6YjIdiRRjgkL"
+  },
+  {
+   "cell_type": "code",
+   "source": [
+    "import tempfile\n",
+    "_, tflite_file_path = tempfile.mkstemp('.tflite') # Path of exported model\n",
+    "mct.exporter.keras_export_model(model=quantized_model, save_model_path=tflite_file_path,\n",
```
Comment (elad-c) [inline tutorials/notebooks/example_keras_imagenet.ipynb:432]: 1. better to just save it locally: replace "tflite_file_path" with the string "qmodel.tflite". Same for the keras export. Then delete the tempfile import
2. this code will fails on the latest MCT release, so better to install mct-nightly

**Review by elad-c — CHANGES_REQUESTED** [review]
1. Add comments for all files, same as in the (effdet tutorial)[https://github.com/sony/model_optimization/blob/main/tutorials/notebooks/example_keras_effdet_lite0.ipynb]
2. Consdier merging consecutive "markdown" cells to a single cell

**Review by elad-c — APPROVED** [review]
